src/libraries/System.Net.Mail/tests/Functional/MailMessageTest.cs

```diff
@@ -146,6 +146,7 @@ public void SubjectAndEncodingTest()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // not passing yet with Value cannot be null.
```

Comment:
The comment suggests that this could be fixed? If so, could this be made into an issue?

Reply:
Changed message.  internal System.Net.Mail.MailWriter not needed in build and the test is trying to load it.